manifests/init.pp

```diff
@@ -107,5 +123,67 @@
     ],
     require     => Package['chocolatey.server'],
   }
-  # technically you may only need IIS_IUSRS but I have not tested this yet.
+
+  # configure chocolatey server settings
+  file { "${_chocolatey_server_location}/web.config":
+    ensure  => file,
+    content => epp('chocolatey_server/web.config.epp', {
+      'allowOverrideExistingPackageOnPush' => $allow_package_override,
+      'apiKey'                             => $apikey,
+      'requireApiKey'                      => $require_apikey,
+    }),
```

Comment:
Just realized this was EPP and not ERB. It requires future parser in 3.5+ or Puppet v4+. I'm not against using EPP, but hoping to allow this to use erb instead for a bit further compatibility. Is there any disadvantage to using ERB?